Lib/string/__init__.py

```diff
@@ -292,34 +288,32 @@ def convert_field(self, value, conversion):
             return ascii(value)
         raise ValueError("Unknown conversion specifier {0!s}".format(conversion))
 
-
-    # returns an iterable that contains tuples of the form:
-    # (literal_text, field_name, format_spec, conversion)
-    # literal_text can be zero length
-    # field_name can be None, in which case there's no
-    #  object to format and output
-    # if field_name is not None, it is looked up, formatted
-    #  with format_spec and conversion and then used
     def parse(self, format_string):
+        """
+        Return an iterable that contains tuples of the form
+        (literal_text, field_name, format_spec, conversion).
+
+        *literal_text* can be zero length.
+        *field_name* can be None, in which case there's no object
+        to format and output; otherwise, it is looked up and
```

Comment:
```suggestion
        *literal_text* can be zero length.

        *field_name* can be None, in which case there's no object
        to format and output; otherwise, it is looked up and
```